# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/bindings/Bindings.conf
# Commit: b55991835aec
# Full Hash: b55991835aecc36d7b8cf86d753c54847bd8b0ec
# Author: Vincent Hilla <vhilla@mozilla.com>
# Date: 2023-12-05 21:29:44
# Regressor Bug: 1589554
# File Overlap Count: 1
# Description:
#   Bug 1589554 - Part 1: Screen Wake Lock boilerplate. r=dom-core,smaug,emilio,edgar
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D189508
# ==============================================================================

diff -r 8c57ecb4b95f -r b55991835aec dom/bindings/Bindings.conf
--- a/dom/bindings/Bindings.conf	Tue Dec 05 15:26:41 2023 +0000
+++ b/dom/bindings/Bindings.conf	Tue Dec 05 15:37:05 2023 +0000
@@ -1028,6 +1028,10 @@
   'nativeType': 'mozilla::dom::TextTrackRegion',
 },
 
+'WakeLock': {
+    'nativeType': 'mozilla::dom::WakeLockJS'
+},
+
 'WebExtensionContentScript': {
     'nativeType': 'mozilla::extensions::WebExtensionContentScript',
 },